static/app/utils/useDismissAlert.tsx

```diff
@@ -0,0 +1,50 @@
+import {useCallback, useEffect, useState} from 'react';
+
+import localStorage from 'sentry/utils/localStorage';
+
+type Opts = {
+  /**
+   * Key in localstorage.
+   * Use a format like: `${organization.id}:my-feature`
+   */
+  key: string;
+
+  /**
+   * Number of days before the dismissal is expired.
+   * After expiration the the user will need to re-dismiss things.
+   */
+  expirationDays?: number;
+};
+
+const MS_PER_DAY = 1000 * 60 * 60 * 24;
+
+function useDismissAlert({expirationDays = Number.MAX_SAFE_INTEGER, key}: Opts) {
```

Comment:
Could we add some description to this? It sounds pretty generic

Reply:
added!